packages/core/resources/icons/icons.json

```diff
@@ -2856,10 +2856,38 @@
     "content": "\\e78a"
   },
   {
-    "name": "New prescription",
+    "name": "Prescription: new",
     "className": "pt-icon-new-prescription",
```

Comment:
This `className` is inconsistent with the `name` now. Do we want to track changing this to `pt-icon-prescription-new` (or maybe supporting both that and `pt-icon-new-prescription`)?

Reply:
please don't change the class name or support multiple names. display names can diverge, that's ok